clients/python/girder_client/cli.py

```diff
@@ -146,9 +155,21 @@ def format_options(self, ctx, formatter):
                    '[default: %s]' % GirderClient.DEFAULT_API_ROOT,
               show_default=True,
               cls=_AdvancedOption)
+@click.option('--no-ssl-verify', is_flag=True, default=False,
+              help='Disable SSL Verification',
+              show_default=True,
+              cls=_AdvancedOption
+              )
+@click.option('--certfile', default=None,
```

Comment:
One other nit, could we change this to `--certificate`? The underlying API supports passing a file *or* a directory containing certificates.